packages/quantic/force-app/main/default/lwc/quanticSourceCitations/__tests__/quanticSourceCitations.test.js

```diff
@@ -0,0 +1,152 @@
+// @ts-ignore
+import {createElement} from 'lwc';
+import QuanticSourceCitations from '../quanticSourceCitations';
+
+const functionsMocks = {
+  mockCitationClickHandler: jest.fn((citationId) => {
+    return citationId;
+  }),
+};
+
+const mockCitations = [
+  {
+    id: '1',
+    title: 'Example title 1',
+    uri: 'https://example.com/',
+    clickUri: 'https://example.com/',
+    permanentid: '1',
+  },
+  {
+    id: '2',
+    title: 'Example title 2',
+    uri: 'https://example.com/',
+    clickUri: 'https://example.com/',
+    permanentid: '2',
+  },
+];
+
+const defaultOptions = {
+  citations: mockCitations,
+  citationClickHandler: functionsMocks.mockCitationClickHandler,
+};
+
+const selectors = {
+  citation: '.citation',
+  citationIndex: '.citation__index',
+};
```

Comment:
If you're going to adopt this pattern, do it all the way ;) 

```suggestion
const selectors = {
  citation: '.citation',
  citationIndex: '.citation__index',
  citationLink: '.citation__link',
};
```